OMMatlab.m

```diff
@@ -569,7 +569,11 @@ function createcsvData(obj, startTime, stopTime)
             obj.csvfile = replace(fullfile(obj.mattempdir,[char(obj.modelname),'.csv']),'\','/');
             fileID = fopen(obj.csvfile,"w");
             %disp(strjoin(fieldnames(obj.inputlist),","));
-            fprintf(fileID,['time,',strjoin(fieldnames(obj.inputlist),","),',end\n']);
+            inputStrings=fieldnames(obj.inputlist);
+            for i = 1:numel(inputStrings)
+                inputStrings_Modified{i,1} = regexprep(inputStrings{i}, '_([^_]*)_', '[$1]');
+            end

```

Comment:
@MohammadHadiAlizadeh we should try to use the `mappednames`  to get the original modelica name when writing to file like we do it for overriding parameters

Reply:
@arun3688 Yes, indeed! 
TY